app/controllers/organizations_controller.rb

```diff
@@ -65,18 +65,17 @@ def demote_to_user
     end
   end
 
-  def deactivate_user
-    user = User.with_discarded.find_by!(id: params[:user_id])
-    raise ActiveRecord::RecordNotFound unless user.has_role?(Role::ORG_USER, current_organization)
-    user.discard!
-    redirect_to user_update_redirect_path, notice: "User has been deactivated."
-  end
-
-  def reactivate_user
-    user = User.with_discarded.find_by!(id: params[:user_id])
+  def remove_user
```

Comment:
I modified this method rather than using the admin roles controller because that you required the super admin role and I thought it did not makes sense to alter that logic